client/packages/purchasing/src/purchase_order/DetailView/LineEdit/PurchaseOrderLineEdit.tsx

```diff
@@ -1,99 +1,224 @@
 import React from 'react';
 import {
-  BasicTextInput,
-  Box,
-  DataTable,
-  Divider,
-  Grid,
-  PurchaseOrderNodeStatus,
+  InputWithLabelRow,
+  ModalGridLayout,
+  // PurchaseOrderNodeStatus,
```

Comment:
TODO note in parent to make it calculate like how it is in the columns right now. Haven't removed columns yet